Formula/govc.rb

```diff
@@ -0,0 +1,21 @@
+class Govc < Formula
+  desc "Command-line tool for VMware vSphere"
+  homepage "https://github.com/vmware/govmomi/tree/master/govc"
+  url "https://github.com/vmware/govmomi/archive/v0.21.0.tar.gz"
+  sha256 "3690bc16e6193ee0d142e8abc1941ad38f841494dd460277722ba3873e0793dc"
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "969b5a81652bcf00427e7024ef4a2198c9dd68dc3b355fdf5e1a178de4e82733" => :mojave
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", "#{bin}/#{name}", "./#{name}"
+  end
+
+  test do
+    system "#{bin}/#{name}", "version"
```

Comment:
Thanks for the PR, the install looks good on the first try 👍. I just have a comment on the testing.

We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.